concordia/templates/transcriptions/asset_detail.html

```diff
@@ -142,7 +142,7 @@ <h2>Tags</h2>
                     {% else %}
                         <p class="help-text primary-text">
                             Only registered users can tag.
-                            <a class="primary-text" href="/account/register/">Register</a> or <a class="primary-text" href="/account/login">Login</a>
+                            <a class="primary-text" href="/account/register/">Register</a> or <a class="primary-text" href="/accounts/login">Login</a>
```

Comment:
We should use URL reversing to make sure these paths are always correct. The login link also will now return them to this page:

```suggestion
                            <a class="primary-text" href="{% url 'registration_register' %}">Register</a> or <a class="primary-text" href="{% url 'login' %}?next={{ request.path|urlencode }}">Login</a>
```